src/main/java/supersymmetry/api/recipes/SuSyRecipeMaps.java

```diff
@@ -301,6 +303,9 @@ public class SuSyRecipeMaps {
     public static final RecipeMap<SimpleRecipeBuilder> POLISHING_MACHINE = new RecipeMap<>("polishing_machine", 1, 1, 2, 1, new SimpleRecipeBuilder(), false)
             .setProgressBar(GuiTextures.PROGRESS_BAR_MIXER, ProgressWidget.MoveType.CIRCULAR)
             .setSound(GTSoundEvents.CENTRIFUGE);
+    public static final RecipeMap<SimpleRecipeBuilder> LARGE_MIXER_RECIPES = new RecipeMap<>("large_mixer", 7, 7, 5, 4, new SimpleRecipeBuilder().EUt(VA[LV]), false)
```

Comment:
no need for so many outputs, 1 fluid and item output

Reply:
ok